core/src/main/resources/oracle/weblogic/deploy/messages/wlsdeploy_rb.properties

```diff
@@ -472,6 +472,10 @@ WLSDPLY-06021=The variables file command line argument {0} was used but the mode
   {1} does not exist : {2}
 WLSDPLY-06022=Discover domain {0}
 WLSDPLY-06023=No domain name found in the domain configuration
+WLSDPLY-06024=Variable file not provided as a command line argument. Automatic replacement of passwords with tokens \
```

Comment:
I think we should brainstorm a few minutes on these messages.  6025 could be a little clearer.

Reply:
done